plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardPage.java

```diff
@@ -314,6 +334,32 @@ protected void onUserdocLinkClicked(final IObservableValue<String> userdocUrlObs
 		new BrowserUtility().checkedCreateExternalBrowser(userdocUrl, OpenShiftCommonUIActivator.PLUGIN_ID,
 				OpenShiftCommonUIActivator.getDefault().getLog());
 	}
+	
+	/**
+	 * Function creates openshift connection and updates pageModel accordingly to pasted oc login command.
+	 * If the login command does not have correct format, then user is informed in error dialog.
+	 * 
+	 */
+	protected void onPasteFromClipboardClicked() {
+		Clipboard clipboard = new Clipboard(Display.getDefault());
+		String clipboardText = (String) clipboard.getContents(TextTransfer.getInstance());
```

Comment:
What if I copied a non text data (ie image) ?

Reply:
What if clipboard content is not String

Reply:
When clipboard content is not String, then clipboard.getContents(TextTransfer.getInstance()) returns null, because it is expecting string(https://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fswt%2Fdnd%2Fpackage-summary.html)